src/apps/Permissions/AppCard.js

```diff
@@ -1,100 +1,95 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import PropTypes from 'prop-types'
-import styled from 'styled-components'
-import { Text, Card, Badge, theme, unselectable } from '@aragon/ui'
+import { Card, Badge, unselectable, GU } from '@aragon/ui'
 import { AppType } from '../../prop-types'
 import { shortenAddress } from '../../web3-utils'
 import AppIcon from '../../components/AppIcon/AppIcon'
 
-class AppCard extends React.PureComponent {
-  static propTypes = {
-    app: AppType.isRequired,
-    onOpen: PropTypes.func.isRequired,
-  }
+const AppBadge = Badge.App
```

Comment:
Maybe we can update to `Tag` with `mode="app"` now?

Reply:
Fixed in 424968e.